webapp/package.json

```diff
@@ -18,10 +18,11 @@
         "react-bootstrap": "1.3.0",
         "react-custom-scrollbars": "4.2.1",
         "react-redux": "7.2.1",
+        "react-select": "3.1.0",
         "redux": "4.0.5"
     },
     "devDependencies": {
-        "@babel/cli": "7.11.6",
+        "@babel/cli": "7.17.10",
```

Comment:
if this is intentional then its okay, otherwise this should not be in PR, if this happened on your local you can keep it there but should not push it